README.md

```diff
@@ -186,6 +186,21 @@ scripts/exec npm update
 # Rebuild and restart the service
 scripts/start --clean
 
+# Dynamic provisioning of Azure Service Bus queues
+The `provision.azure.yaml` manifest file is used to declare Azure Service Bus queues that will be provisioned for both a deployed Pull Request and for integration tests running in CI.
+
+As this service requires three queues, the structure of the file should be:
+
+These queues will be automatically removed when the Pull Request is closed.
```

Comment:
Small nitpick, I think it would read better if the line "these queues..." came after the example resources definition